_stbt/core.py

```diff
@@ -720,22 +720,11 @@ def timestamp(self):
             return int(self.time * 1e9)
 
 
-def new_device_under_test_from_config(
-        gst_source_pipeline=None, gst_sink_pipeline=None, control_uri=None,
-        save_video=False, restart_source=None, transformation_pipeline=None):
-    from _stbt.control import uri_to_remote
+def new_device_under_test(
+        gst_source_pipeline, gst_sink_pipeline, control_uri, save_video=False,
+        restart_source=False, transformation_pipeline='identity'):
 
-    if gst_source_pipeline is None:
-        gst_source_pipeline = get_config('global', 'source_pipeline')
-    if gst_sink_pipeline is None:
-        gst_sink_pipeline = get_config('global', 'sink_pipeline')
-    if control_uri is None:
-        control_uri = get_config('global', 'control')
-    if restart_source is None:
-        restart_source = get_config('global', 'restart_source')
-    if transformation_pipeline is None:
-        transformation_pipeline = get_config('global',
-                                             'transformation_pipeline')
```

Comment:
I do make use of this sometimes from Jupyter notebooks and the like when I'm experimenting.  If possible I'd prefer the  defaults to be removed from stbt-run rather than here.

Reply:
OK, I've dropped this change, it wasn't really related anyway.